src/styles/item.scss

```
@@ -0,0 +1,34 @@
+.item {
+  position: relative;
+  display: block;
+  font-size: 32px;
+  line-height: 33.75px;
+  letter-spacing: -0.6%;
+  color: white;
```

```suggestion
  position: relative;
  
  display: block;
  
  font-size: 32px;
  line-height: 33.75px;
  letter-spacing: -0.6%;
  color: white;
```

відформатуй весь код правильно